cmd/context/context.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2021 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package context
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/okteto/okteto/cmd/utils"
+	"github.com/okteto/okteto/pkg/analytics"
+	okContext "github.com/okteto/okteto/pkg/cmd/context"
+	k8Client "github.com/okteto/okteto/pkg/k8s/client"
+	"github.com/okteto/okteto/pkg/log"
+	"github.com/spf13/cobra"
+)
+
+type ContextOptions struct {
+	Token           string
+	isOktetoCluster bool
```

Comment:
Keep in mind that something like "localhost" or "foo" are valid DNS values as well as a valid URLs, and it can be used as okteto endpoint. So we might want to consider a way for the user to specify if the value passed is a URL or a kubernetes context.

Reply:
We might want to add a way to specify it for those corner cases where a URL === kubernetes context name, but the default way should be simple and it can be inferred for most cases. Are we taking the kubernetes context as the preference? If a context `foo` doesn't exist, it's treated as a URL, for example.

Reply:
yeah, I think that's a good approach @rlamana .